HISTORY.rst

```diff
@@ -3,6 +3,15 @@
 Release History
 ---------------
 
+**Unreleased**
++++++++++++++++++++
+
+- The behavior of `SessionRedirectMixin` was slightly altered.
+  `resolve_redirects` will now detect a redirect by calling
+  `get_redirect_target(response)` instead of directly
+  querying `Response.is_redirect` and `Response.headers['location'].  Advanced
+  users will be able to process malformed redirects more easily.
```

Comment:
reStructuredText uses double-backticks to force monospace font, not single-backticks.